core/src/test/java/nl/nn/adapterframework/jdbc/dbms/OracleToH2TranslatorTest.java

```diff
@@ -0,0 +1,92 @@
+package nl.nn.adapterframework.jdbc.dbms;
+
+import static org.junit.Assert.assertEquals;
+
+import java.sql.SQLException;
+
+import nl.nn.adapterframework.jdbc.JdbcException;
+
+import org.junit.Test;
+
+public class OracleToH2TranslatorTest {
+
+	@Test
+	public void testConvertQuerySelect() throws JdbcException, SQLException {
+		String query = " SELECT FIELD1, FIELD2, DECODE(FIELD3,'Y','true','N','false',NULL,'true') AS FIELD3, LISTAGG(FIELD4, ' + ') WITHIN GROUP (ORDER BY FIELD4) AS FIELD4 FROM TABLE1 \n\t  \n  GROUP BY FIELD1, FIELD2, FIELD3, FIELD4 ORDER BY FIELD4, FIELD2  \n";
```

Comment:
Het is voor dit soort testen prettig om het verwachte resultaat meteen als String expected="..." te definieren onder de input. Als ze dan onder elkaar staan zie je goed het verschil dat er wel of niet moet zijn.

Reply:
https://github.com/ibissource/iaf/pull/251/commits/650e4e3f83124bda43fe17e7693015c12dafc3e5